pkg/sdk/testint/secrets_gen_integration_test.go

```diff
@@ -0,0 +1,673 @@
+		require.NoError(t, err)
+		require.Contains(t, returnedSecrets, *secretOAuthClientCredentials)
+		require.Contains(t, returnedSecrets, *secretOAuthAuthorizationCode)
+		require.Contains(t, returnedSecrets, *secretBasicAuthentication)
+		require.Contains(t, returnedSecrets, *secretGenericString)
+
+		returnedSecrets, err = client.Secrets.Show(ctx, sdk.NewShowSecretRequest().WithIn(sdk.In{Schema: id.SchemaId()}))
+		require.NoError(t, err)
+		require.Contains(t, returnedSecrets, *secretOAuthClientCredentials)
+		require.Contains(t, returnedSecrets, *secretOAuthAuthorizationCode)
+		require.Contains(t, returnedSecrets, *secretBasicAuthentication)
+		require.Contains(t, returnedSecrets, *secretGenericString)
+	})
+
+	t.Run("Show: SecretWithGenericString with In", func(t *testing.T) {
+		secret, id := createSecretWithGenericString(t, "foo", nil)
+
+		returnedSecrets, err := client.Secrets.Show(ctx, sdk.NewShowSecretRequest().WithIn(sdk.In{Account: sdk.Pointer(true)}))
+		require.NoError(t, err)
+		require.Contains(t, returnedSecrets, *secret)
+
+		returnedSecrets, err = client.Secrets.Show(ctx, sdk.NewShowSecretRequest().WithIn(sdk.In{Database: id.DatabaseId()}))
+		require.NoError(t, err)
+		require.Contains(t, returnedSecrets, *secret)
+
+		returnedSecrets, err = client.Secrets.Show(ctx, sdk.NewShowSecretRequest().WithIn(sdk.In{Schema: id.SchemaId()}))
+		require.NoError(t, err)
+		require.Contains(t, returnedSecrets, *secret)
+	})
+
+	t.Run("ShowByID", func(t *testing.T) {
+		_, id := createSecretWithGenericString(t, "foo", nil)
+
+		secret, err := client.Secrets.ShowByID(ctx, id)
+		require.NoError(t, err)
+		assertSecret(t, secret, id, "GENERIC_STRING", "")
+	})
+
+	t.Run("Describe", func(t *testing.T) {
+		_, id := createSecretWithGenericString(t, "foo", func(req *sdk.CreateWithGenericStringSecretRequest) {
+			req.WithComment("Lorem ipsum")
+		})
+
+		details, err := client.Secrets.Describe(ctx, id)
+		require.NoError(t, err)
+
+		assertSecretDetails(details, secretDetails{
+			Name:       id.Name(),
+			Comment:    "Lorem ipsum",
+			SecretType: "GENERIC_STRING",
+		})
+	})
+}
+
+func TestInt_SecretsShowWithIn(t *testing.T) {
+	client := testClient(t)
+	ctx := testContext(t)
+
+	cleanupSecret := func(t *testing.T, id sdk.SchemaObjectIdentifier) func() {
+		t.Helper()
+		return func() {
+			err := client.Secrets.Drop(ctx, sdk.NewDropSecretRequest(id).WithIfExists(true))
+			require.NoError(t, err)
+		}
+	}
+
+	createSecretWithGenericString := func(t *testing.T, id sdk.SchemaObjectIdentifier, secretString string) *sdk.Secret {
+		t.Helper()
+		request := sdk.NewCreateWithGenericStringSecretRequest(id, secretString)
+		err := client.Secrets.CreateWithGenericString(ctx, request)
+		require.NoError(t, err)
+		t.Cleanup(cleanupSecret(t, id))
+
+		secret, err := client.Secrets.ShowByID(ctx, id)
+		require.NoError(t, err)
+
+		return secret
+	}
+
+	t.Run("Show with In - same id in different schemas", func(t *testing.T) {
```

Comment:
You have a lot of SHOW tests, isn't this case covered in one of them? You can also test it in the main function, no need to have it in `TestInt_SecretsShowWithIn`.

Reply:
You're right, its kinda of a duplication so it will be removed